readme.md

```diff
@@ -66,6 +66,46 @@ erDiagram
       int permission_id FK
    }
 
+   user_groups_to_projects {
```

Comment:
We don't need these newly added tables, they previously were removed 🤔 Instead we need to have a similar logic to user_groups, but instead user_groups we have project_groups and instead permissions we have project_permissions. We need to describe tables added in this pr